ordered_model/models.py

```diff
@@ -80,8 +80,8 @@ def move(self, direction, qs=None):
             _("The method move() is deprecated and will be removed in the next release."),
             DeprecationWarning
         )
-        if direction == 'up':
-            self.up()
+        if direction in ['up', 'down', 'top', 'bottom']:
+            getattr(self, direction)()
         else:
```

Comment:
Then `move_view` should call `up` and `down` also? If yes, it would be reasonable to do change for all move methods together in some other pull request.